File: apps/claimly-server/src/APIs/Claim/Dtos/ClaimWhereInput.cs
namespace Claimly.APIs.Dtos;

public class ClaimWhereInput
{
    public double? Amount { get; set; }

    public string? ClaimStatus { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? Customer { get; set; }

    public DateTime? Date { get; set; }

    public string? Description { get; set; }

    public string? Id { get; set; }

    public string? Policy { get; set; }

    public DateTime? UpdatedAt { get; set; }
}

File: apps/claimly-node-js-admin/src/api/claim/ClaimWhereInput.ts
import { FloatNullableFilter } from "../../util/FloatNullableFilter";
import { ClaimStatusWhereUniqueInput } from "../claimStatus/ClaimStatusWhereUniqueInput";
import { CustomerWhereUniqueInput } from "../customer/CustomerWhereUniqueInput";
import { DateTimeNullableFilter } from "../../util/DateTimeNullableFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { StringFilter } from "../../util/StringFilter";

export type ClaimWhereInput = {
  amount?: FloatNullableFilter;
  claimStatus?: ClaimStatusWhereUniqueInput;
  customer?: CustomerWhereUniqueInput;
  date?: DateTimeNullableFilter;
  description?: StringNullableFilter;
  id?: StringFilter;
};
